src/widgets.rs

```diff
@@ -24,8 +24,12 @@
 mod barchart;
 pub mod block;
 mod borders;
-#[cfg(feature = "widget-calendar")]
+#[cfg(any(feature = "widget-calendar", feature = "widget-calendar-chrono"))]
 pub mod calendar;
+#[cfg(all(feature = "widget-calendar", feature = "widget-calendar-chrono"))]
+compile_error!(
+    "These two feature flags are actually conflicting. You are supposed to only use one of them"
+);
```

Comment:
This is horrible for trying to check as much code as possible with `--all-features`. (I know it's already a problem on Windows, but we don't need to make this even harder)

Reply:
Yeah, I agree here. I wonder if there's a way we can support both time and chrono in the calendar widget?

I want to soften the language here a bit to make sure that @InnocentZero doesn't get offended by this being called "horrible". I think Ed means this as a shorthand for this causing unintended consequences in a way that would be frustrating when developing. We often will use the `--all-features` flag to make it easy to make changes that build all the code. Taking away that ability makes it more difficult to develop the library.